flock.go

```diff
@@ -47,19 +47,21 @@ func (f *Flock) Close() error {
 	return f.Unlock()
 }
 
-// Path is a function to return the path as provided in NewFlock().
+// Path returns the path as provided in NewFlock().
 func (f *Flock) Path() string {
 	return f.path
 }
 
-// Locked is a function to return the current lock state (locked: true, unlocked: false).
+// Locked returns the lock state (locked: true, unlocked: false).
+// Warning: by the time you use the returned value, the state may have changed.
```

Comment:
So that the godoc renders it on its own line, to make it more prominent, can you have the comment look like:

```Go
// Locked returns the lock state (locked: true, unlocked: false).
//
// Warning: by the time you use the returned value, the state may have changed.
```